15/03/2018 02:18:05 | SyncInvokeChargerLotriPourCalcul | Debut 15/03/2018 02:18:05

15/03/2018 02:18:09 | SyncInvokeChargerLotriPourCalcul | Debut 15/03/2018 02:18:09

15/03/2018 10:38:39 | SyncInvokeChargerLotriPourCalcul | Debut 15/03/2018 10:38:39

15/03/2018 10:38:44 | SyncInvokeChargerLotriPourCalcul | Debut 15/03/2018 10:38:44

15/03/2018 10:45:07 | SyncInvokeChargerLotriPourCalcul | Debut 15/03/2018 10:45:07

15/03/2018 10:45:09 | SyncInvokeChargerLotriPourCalcul | Debut 15/03/2018 10:45:09

15/03/2018 10:52:47 | SyncInvokeChargerLotriPourCalcul | Debut 15/03/2018 10:52:47

15/03/2018 10:52:58 | SyncInvokeChargerLotriPourCalcul | Debut 15/03/2018 10:52:58

15/03/2018 11:45:09 | SyncInvokeChargerLotriPourCalcul | Debut 15/03/2018 11:45:09

15/03/2018 11:45:40 | ChargerLotriPourCalcul | SPX_FAC_CHARGERLOTPOURCALCUL:Une erreur de niveau transport s'est produite lors de la réception des résultats du serveur. (provider: TCP Provider, error: 0 - La connexion réseau a été arrêtée par le système local.)

15/03/2018 11:47:21 | SyncInvokeChargerLotriPourCalcul | Debut 15/03/2018 11:47:21

15/03/2018 11:47:25 | SyncInvokeChargerLotriPourCalcul | Debut 15/03/2018 11:47:25

15/03/2018 11:48:19 | LogException | Une erreur de niveau transport s'est produite lors de la réception des résultats du serveur. (provider: Session Provider, error: 19 - La connexion physique ne peut pas être utilisée)

15/03/2018 11:48:19 | CalculeDuLotGeneral | Une erreur s'est produite lors de l'exécution de la définition de la commande. Pour plus de détails, consultez l'exception interne.

15/03/2018 11:53:49 | SyncInvokeChargerLotriPourCalcul | Debut 15/03/2018 11:53:49

15/03/2018 11:53:52 | SyncInvokeChargerLotriPourCalcul | Debut 15/03/2018 11:53:52

